### PR TITLE
XSS fix

### PR DESCRIPTION
**Pull request by rgksugan**
The current code missed the code to fix the css from an older pull request.

Added it back.

### CODE REVIEW

**Comment by rgksugan** [comment]
https://github.com/dibari/angular-ellipsis/issues/39

File: src/angular-ellipsis.js [hunk 1]
```diff
@@ -52,74 +52,93 @@ angular.module('dibari.angular-ellipsis',[])
...
+								element.text(binding).html(element.html() + appendString);
+							}
 							//Set data-overflow on element for targeting
 							element.attr('data-overflowed', 'true');
 
 							// Set complete text and remove one word at a time, until there is no overflow
-							for ( ; i < bindArrayStartingLength; i++) {
+							for (; i < bindArrayStartingLength; i++) {
 								bindArray.pop();
 								element.html(bindArray.join(ellipsisSeparator) + appendString);
```
Comment (sod) [inline src/angular-ellipsis.js]: This now executes twice? <- first call

File: src/angular-ellipsis.js [hunk 2]
```diff
@@ -52,74 +52,93 @@ angular.module('dibari.angular-ellipsis',[])
...
 
-								if ((scope.useParent?element.parent()[0]:element[0]).scrollHeight < initialMaxHeight || isOverflowed(element, scope.useParent) === false) {
+								if (isHtml) {
+									element.html(bindArray.join(ellipsisSeparator) + appendString);
```
Comment (sod) [inline src/angular-ellipsis.js]: <- second call?

**Comment by rgksugan** [comment]
@sod fixed it

**Pull request merged**